app/src/main/res/layout/quicksettings_permissions.xml

```diff
@@ -136,12 +136,38 @@
         android:layout_width="wrap_content"
         android:layout_height="@dimen/quicksettings_item_height"
         android:visibility="gone"
-        app:layout_constraintBottom_toTopOf="@id/mediaKeySystemAccessStatus"
+        app:layout_constraintBottom_toTopOf="@id/storageAccessStatus"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@id/persistentStorageLabel"
         tools:text="@string/preference_option_phone_feature_blocked"
         tools:visibility="visible" />
 
+    <TextView
+        android:id="@+id/storageAccessLabel"
+        style="@style/QuickSettingsText.Icon"
+        android:layout_width="0dp"
+        android:layout_height="@dimen/quicksettings_item_height"
+        app:drawableStartCompat="@drawable/ic_storage"
```

Comment:
Nit: Maybe we could use a [cookie icon](https://design.firefox.com/icons/viewer/#cookie)?  :)